apps/src/lab2/lab2Redux.ts

```diff
@@ -16,6 +16,7 @@ import {
   ProjectManagerStorageType,
   ProjectSources,
   LevelProperties,
+  ProjectLevelData,
```

Comment:
Just cuiours - why did you add this? I don't see it being used in file.

Reply:
Good catch! Removed